bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2Epics.scala

```diff
@@ -99,26 +100,27 @@ object Flamingos2Epics {
 
 }
 
-object Flamingos2EpicsInitializer {
+object Flamingos2Epics extends EpicsSystem {
+  
   val Log = Logger.getLogger(getClass.getName)
   val CA_CONFIG_FILE = "/Flamingos2.xml"
+  
+  // Ugly use of null. But if it is used before initialization, then the application deserves to crash.
+  var instance: Flamingos2Epics = null
```

Comment:
It is like that. I use the "instance" var only to access the only instance of the class. Maybe I should use a method to access the only instance, that checks if it exists and creates it otherwise.